### PR TITLE
improve annotator example and fix test

### PR DESCRIPTION
**Pull request by fouad-j**

### CODE REVIEW

File: code_samples/simple_language_plugin/src/com/simpleplugin/SimpleAnnotator.java [hunk 1]
```diff
@@ -1,37 +1,41 @@
 package com.simpleplugin;
```
Comment (JohnHake) [inline code_samples/simple_language_plugin/src/com/simpleplugin/SimpleAnnotator.java]: Thank you for contributing!
1) Can you describe the defect your changes fix? (Or the feature it adds?)
2) Would you please push a set of changes to SimpleAnnotator.java based on the current revision of the file? There were substantial changes to the annotate method checked-in on 11-Feb-2020. The baseline you have used here predates those changes.

File: code_samples/simple_language_plugin/testData/AnnotatorTestData.java [hunk 2]
```diff
@@ -2,6 +2,6 @@
   public static void main(String[] args) {
     System.out.println("simple:website");
     System.out.println("simple:key with spaces");
-    System.out.println("simple:<error descr="Unresolved property">websit"</error>);
+    System.out.println("simple:<error descr=\"Unresolved property\">website</error>");
```
Comment (JohnHake) [inline code_samples/simple_language_plugin/testData/AnnotatorTestData.java]: The correct statement is:
`System.out.println("simple:<error descr="Unresolved property">websit</error>");`

See [Testing Highlighting](https://www.jetbrains.org/intellij/sdk/docs/basics/testing_plugins/testing_highlighting.html)

Comment (fouad-j) [reply code_samples/simple_language_plugin/testData/AnnotatorTestData.java:5]: for JAVA code you need to escape all "(quotation mark) into (quotation marks)""
you can try to run tests with existing code, they will not work

File: code_samples/simple_language_plugin/src/com/simpleplugin/SimpleAnnotator.java (hunk 1 above)
Comment (fouad-j) [reply code_samples/simple_language_plugin/src/com/simpleplugin/SimpleAnnotator.java:1]: Hello,
1.
- In the first commit I improved the code by : 
  * removing some indentation level
  * doing some extract and call to remove duplicate code
- In the second commit I fixed annotator test
2. I'll rebase my fork

**Pull request closed without merging**